sky/sdk/example/game/lib/particle_system.dart

```diff
@@ -262,19 +322,55 @@ class ParticleSystem extends Node {
       rects.add(rect);
 
       // Color
-      Color particleColor;
-      if (particle.colorSequence != null) {
-        particleColor = particle.colorSequence.colorAtPosition(particle.colorPos);
+      if (particle.simpleColorSequence != null) {
+        Color particleColor = new Color.fromARGB(
```

Comment:
It's silly that Color doesn't have a constructor to do this for you.  I could even imagine a constructor which took a Float32List.